src/ansys/dpf/composites/composite_model.py

```diff
@@ -49,17 +49,23 @@ class CompositeScope:
         Time or frequency. You can use the
         :func:`CompositeModel.get_result_times_or_frequencies` method
         to list the solution steps.
+    named_selections:
+        List of element sets.
+        Use `composite_model.get_mesh().available_named_selections` to list
+        all named selections.
 
     Notes
     -----
-    If both the ``elements`` and ``plies`` parameters are set, the final element
-    scope is the intersection of the two.
+    If more than one scope (``elements``, ``named_selections`` and ``plies``)
+    is set, then the final element scope is the intersection
```

Comment:
Is this correct? My understanding is that we take the union of elements and named_selections and then intersect it with plies.

Reply:
I was wrong. There is an intersection of elements and named selections